.env.example

```diff
@@ -0,0 +1,12 @@
+# NOTE: Using a .env file is optional. The below values are defauled in
+# docker-compose.yml as well, and `docker compose up` works without a .env file.
+
+# The port at which VAST can be accessed. Use
+# `vast -e :42000 <command>` to interact with VAST.
+VAST_PORT=42000
+
+# The Git revision of VAST to use
+# when building VAST images from source. Run `git rev-parse HEAD` to get the current
+# revision, or use the special tag `latest` to refer to the latest upstream
+# revision.
+VAST_VERSION=latest
```

Comment:
There's a missing newline at EOF here.

Reply:
Latest commit fixes this. Thanks for pointing this.